ush/templates/FV3LAM_wflow.xml

```diff
@@ -307,7 +307,140 @@ MODULES_RUN_TASK_FP script.
 ************************************************************************
 ************************************************************************
 -->
-  <task name="&GET_EXTRN_ICS_TN;" cycledefs="initial" maxtries="{{ maxtries_get_extrn_ics }}">
+{%- if do_enscontrol %}
+ <metatask name="run_ensemble">
+ <var name="{{ ensmem_indx_name }}">
+  {%- for m in range(1, 2) -%}
+  {%- set fmtstr=" %0"~ndigits_ensmem_names~"d" -%}
+  {{- fmtstr%m -}}
+ {%- endfor %} </var>
+
+  <task name="&GET_EXTRN_ICS_TN;{{ uscore_ensmem_name }}" cycledefs="initial" maxtries="{{ maxtries_get_extrn_ics }}">
```

Comment:
The task &GET_EXTRN_ICS_TN;{{ uscore_ensmem_name } and &GET_EXTRN_LBCS_TN;{{ uscore_ensmem_name } appear twice separately. What the differences between each appearance? Can they be combined into one?